ISISComputingGroup/ibex_gui
# Fix script generator exception on entering data

### Description of work

Offset editing support column from viewer table focus column and coax value into range

To test:

- Tab across table whilst entering values to affect the validity of the row
- Copy and paste rows (selecting both editable and non-editable rows)
- Check validity on inserted rows are correct
- Check log doesn't have any index out of bounds errors

### Ticket

https://github.com/ISISComputingGroup/IBEX/issues/6204

### Acceptance criteria

*List the acceptance criteria for the PR. The aim is provide information to help the reviewer*

### Unit tests

*Give an overview of unit tests you have added or modified, if applicable. The aim is provide information to help the reviewer*

### System tests

*Mention any automated tests or manual tests that you have added or modified, if applicable. The aim is provide information to help the reviewer*

### Documentation
*Highlight and provide a link to any additions or changes to the documentation, if applicable. The aim is provide information to help the reviewer*

---

#### Code Review


### Final Steps
